pybossa/api/__init__.py

```diff
@@ -707,7 +707,8 @@ def get_service_request(task_id, service_name, major_version, minor_version):
         if isinstance(service, dict):
             url = '{}/{}/{}/{}'.format(proxy_service_config['uri'], service_name, major_version, minor_version)
             headers = service.get('headers')
-            ret = requests.post(url, headers=headers, json=payload['data'])
+            ssl_cert = current_app.config.get('SSL_CERT_PATH')
+            ret = requests.post(url, headers=headers, json=payload['data'], verify=ssl_cert if ssl_cert else True)
```

Comment:
This could be
```python
ssl_cert = current_app.config.get('SSL_CERT_PATH', False)
ret = requests.post(url, headers=headers, json=payload['data'], verify=ssl_cert)
```

Reply:
@dchhabda `verify` is `True` [by default](https://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification) so this would change existing behavior for missing certs. I will change the first line to:
```
ssl_cert = current_app.config.get('SSL_CERT_PATH', True)
```

Reply:
for missing certs path in configs, IMO `ssl_cert` would be set to `False` which would superimpose default `verify=True` behavior. Let's connect offline if needed. Thanks!